timers.js

```diff
@@ -1,4 +1,30 @@
+$(document).on("ready", function(){
 
+  heading = $("#timer");
+  strtBtn = $("#start");
+  stopBtn = $("#pause");
+  restBtn = $("#reset");
+  seconds = 0;
+  rest = function(){ clearInterval(timr);};
 
+  function StpWtch (){
+    seconds ++;
+    heading.html("Elapsed Time: " + seconds);
+  };
 
+  contain = function(){
+  timr = setInterval(StpWtch, 1000);
+  };
 
+  function reset (){
+    heading.html("Stop Watch");
+    seconds = 0;
+    rest();
+  }
+
+  strtBtn.on("click", contain);
```

Comment:
If I click on the start button 10 times, it will create 10 different timers. Is there a way to prevent this from happening (i.e., only one timer can be running at a given time)?